Formula/php@7.2.rb

```diff
@@ -16,7 +16,7 @@ class PhpAT72 < Formula
 
   keg_only :versioned_formula
 
-  deprecate! date: "2020-11-30", because: :versioned_formula
+  disable! date: "2021-11-30", because: :deprecated_upstream
```

Comment:
Probably should keep it deprecated too.

Reply:
Yup, there's no need for that anymore. A future `disable!` date deprecates the formula until the disable date. We also have a workflow that runs regularly that deletes disabled formulae one year after the disable date.

Reply:
Ah I missed that change in `brew`.